src/plugins/RedPacket/types.ts

```diff
@@ -151,7 +153,6 @@ export interface RedPacketJSONPayload {
     total: string
     creation_time: number
     duration: number
-    network?: EthereumNetwork
```

Comment:
Does change the schema is possible?

Reply:
No, change the schema doesn't resolve the problem. This object is posted on to the web. It's an exchange format. If you change the format, it will break older version Maskbook. It also creates a new format that not compatible with Red Packet implementation in the Tessercube.